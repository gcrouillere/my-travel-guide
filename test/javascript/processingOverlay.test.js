import React from 'react';
import ReactDOM from 'react-dom';
import { shallow, configure, mount } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { MemoryRouter } from 'react-router'
import renderer from 'react-test-renderer';

import ProcessingOverlay from '../../app/javascript/packs/articleForm/photoForm/processingOverlay'

describe('processingOverlay test suite', () => {
  it('renders component', () => {
    let processingOverlayTree = renderer.create(<ProcessingOverlay processing={false}/>).toJSON()
    expect(processingOverlayTree).toMatchSnapshot()
  })
})
