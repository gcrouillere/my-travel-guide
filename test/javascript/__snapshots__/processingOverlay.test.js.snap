// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`processingOverlay test suite renders component 1`] = `
<div
  className="processingOverlay inactive"
>
  <button
    className="btn btn-dark"
    disabled={true}
    type="button"
  >
    <span
      aria-hidden="true"
      className="spinner-border spinner-border-sm"
      role="status"
    />
    Processing ...
  </button>
</div>
`;
